commit fd3a539da52c5d1fdd67494509dc53232aa6ee4a (refs/changes/64/3064/3)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-12-11T20:12:15+00:00 (1 year, 10 months ago)
    
    MANTA-3514 electric-moray delete old version of empty array batch test

diff --git a/test/batch.test.js b/test/batch.test.js
index da78fcb..2bd08e2 100644
--- a/test/batch.test.js
+++ b/test/batch.test.js
@@ -441,12 +441,6 @@ test('same key transforms to different values', function (t) {
 });
 
 
-test('bad requests array: empty array', function (t) {
-    runBadBatch(this.client, t, [],
-        'must specify an array with at least one request');
-});
-
-
 test('unsupported batch operation: "update"', function (t) {
     var key = path.join('/', uuid.v4(), 'stor', uuid.v4());
     var requests = [
